library/src/scala/tasty/Reflection.scala

```diff
@@ -1353,6 +1353,8 @@ trait Reflection extends reflect.Types { reflectSelf: CompilerInterface =>
     def of[T <: AnyKind](using qtype: scala.quoted.Type[T]): Type =
       qtype.asInstanceOf[scala.internal.quoted.Type[TypeTree]].typeTree.tpe
 
+    def ofErasedClass(clazz: Class[_]): Type =
```

Comment:
```suggestion
    def typeConstructorOf(clazz: Class[_]): Type =
```

Reply:
You should also update the following code https://github.com/lampepfl/dotty/blob/07fae04ef13d1e9d18d5661ee58429966ee1a0fb/library/src-bootstrapped/scala/quoted/Liftable.scala#L59-L67

```diff
  given ClassLiftable[T] as Liftable[Class[T]] = new Liftable[Class[T]] {
    def toExpr(x: Class[T]) = {
-      val qctx1 = scala.internal.tasty.CompilerInterface.quoteContextWithCompilerInterface(qctx)
-      import qctx1.tasty._
-      val tpe = qctx1.tasty.Type_ofErasedClass(x)
-      Ref(defn.Predef_classOf).appliedToType(tpe).seal.asInstanceOf[Expr[Class[T]]]
+      import qctx.tasty._
+      Ref(defn.Predef_classOf).appliedToType(Type.typeConstructorOf(x)).seal.asInstanceOf[Expr[Class[T]]]
    }
  }
```

Reply:
Roger that.  I’ll fix with these changes.